src/control/cmd/dmg/storage.go

```diff
@@ -184,6 +187,46 @@ func (cmd *storageFormatCmd) Execute(args []string) error {
 	return resp.Errors()
 }
 
+// storageReformatCmd is the struct representing the command to reformat storage
+// on a stopped DAOS system.
+type storageReformatCmd struct {
```

Comment:
Thanks for making this change. We may still need to iterate in future PRs but I think this grouping makes more sense from a UX perspective.